src/validation.cpp

```diff
@@ -4859,7 +4859,9 @@ void ChainstateManager::CheckBlockIndex()
                // For testing, allow transaction counts to be completely unset.
                || (pindex->nChainTx == 0 && pindex->nTx == 0)
                // For testing, allow this nChainTx to be unset if previous is also unset.
```

Comment:
Yes, the entire assert block is wrong. When a snapshot is loaded, the `pindex->nTx` value is set to 1. So I'm not really sure what these exceptions would be allowing through.

But since this issue affects the v26 release, I've tried to keep the code delta as small and minimally impactful as possible.

Reply:
Yes, because the genesis block.